app/components/agenda/agendaitem/agendaitem-decision.js

```diff
@@ -149,6 +150,13 @@ export default class AgendaitemDecisionComponent extends Component {
           piece
         );
       }
+      let reportToUpdate = this.report;
+      while (reportToUpdate) {
+        await this.pieceAccessLevelService.strengthenAccessLevelToInternRegering(
+          reportToUpdate
+        );
+        reportToUpdate = await reportToUpdate.previousPiece
+      }
```

Comment:
Is the `while` loop necessary? `strengthenAccessLevelToInternRegering` should already update the previous versions.

Reply:
see my comment. I think this entire block should go.
Report access levels stay unchanged (but there should be a difference between the original report and a BIS report)